### PR TITLE
send delete events

### PR DESCRIPTION
**Pull request by qlibin**
Fix for #6

### CODE REVIEW

**Comment by ePaul** [comment]
I did not try it, but it looks good.

File: src/main/java/org/zalando/tarbelaproducer/service/EventLogWriter.java [hunk 1]
```diff
@@ -30,4 +30,16 @@
...
+     * Suppose you want to remove an object from a database and also you want to fire an event
+     * about this object removal. Then you can call this method in the same transaction.
+     * This method will serialize a payload object and will store an event with this payload
+     * in a database and Tarbela will later read the event via the Publisher API (/events endpoint)
+     * to publish this event in an event sink
+     * @param payload some POJO that can be serialized into JSON (required parameter)
+     * @param flowId Optional parameter to provide an X-Flow-ID value
+     */
+    @Transactional
+    void fireDeleteEvent(EventPayload payload, String flowId);
```
Comment (raphaelbauer) [inline src/main/java/org/zalando/tarbelaproducer/service/EventLogWriter.java]: nice... if the flowId is optional I'd create two methods. One

void fireDeleteEvent(EventPayload payload, String flowId);

void fireDeleteEvent(EventPayload payload);

Then it is clear that flowId might be optional... :)

**Comment by raphaelbauer** [comment]
👍

File: src/main/java/org/zalando/tarbelaproducer/service/EventLogWriter.java (hunk 1 above)
Comment (ePaul) [inline src/main/java/org/zalando/tarbelaproducer/service/EventLogWriter.java]: Alternatively a `@Nullable` annotation here.

**Comment by sovereign36** [comment]
👍

**Comment by qlibin** [comment]
👍

**Pull request merged**